core/src/main/scala/kafka/tools/DumpLogSegments.scala

```diff
@@ -257,8 +257,11 @@ object DumpLogSegments {
             }
             lastOffset = record.offset
 
-            print(s"$RecordIndent offset: ${record.offset} ${batch.timestampType}: ${record.timestamp} " +
-              s"keysize: ${record.keySize} valuesize: ${record.valueSize}")
+            print(s"$RecordIndent offset: ${record.offset} baseOffset: ${batch.baseOffset} lastOffset: ${batch.lastOffset} baseSequence: ${} ${batch.baseSequence}" +
+             s" lastSequence: ${batch.lastSequence} producerEpoch: ${batch.producerEpoch} partitionLeaderEpoch: ${batch.partitionLeaderEpoch} position: ${validBytes} " +
+             s" ${batch.timestampType}: ${record.timestamp} isvalid: ${record.isValid}"+
+             s" keysize: ${record.keySize} valuesize: ${record.valueSize} size: ${batch.sizeInBytes} magic: ${batch.magic} " +
```

Comment:
isvalid -> "isValid"
keysize -> "keySize"
valuesize -> "valueSize"
compresscodec -> "compressionType"

Reply:
@chia7712  Have seperated the record details from batch details. Also changed the entries to camel case. Please review this. Thanks.